helper/communicator/config.go

```diff
@@ -175,8 +175,8 @@ type SSH struct {
 	SSHLocalTunnels []string `mapstructure:"ssh_local_tunnels"`
```

Comment:
Don't know what these means but their documentation is empty. I would like to add something here if anyone can tell me the best description.

Reply:
Let's leave them empty for now and document them in a separate PR. I'll have to read the commit where they were added to get context.